api/pkg/apis/v1alpha1/providers/target/staging/staging.go

```diff
@@ -86,22 +86,22 @@ func (i *StagingTargetProvider) Get(ctx context.Context, deployment model.Deploy
 	ctx, span := observability.StartSpan("Staging Target Provider", ctx, &map[string]string{
 		"method": "Get",
 	})
-	sLog.Infof("  P (Staging Target): getting artifacts: %s - %s, traceId: %s", deployment.Instance.Spec.Scope, deployment.Instance.Spec.Name, span.SpanContext().TraceID().String())
+	sLog.Infof("  P (Staging Target): getting artifacts: %s - %s, traceId: %s", deployment.Instance.ObjectMeta.Namespace, deployment.Instance.ObjectMeta.Name, span.SpanContext().TraceID().String())
 
 	var err error
 	defer observ_utils.CloseSpanWithError(span, &err)
 
-	scope := deployment.Instance.Spec.Scope
-	if scope == "" {
-		scope = "default"
+	namespace := deployment.Instance.ObjectMeta.Namespace
```

Comment:
deployment.Instance.Spec.Scope and deployment.Instance.ObjectMeta.Namespace are different. Spec.Scope spcifies where the deployed objects are to be placed; ObjectMetadata.Namespace is where the instance object itself belongs.

Reply:
Hi, Haishi. Thanks for your comments. Do you think we need to keep staging history in the same namespace with instances or with the artifact? I thought we need to keep all related symphony objects in the same namespace so I changed the namespace to where instances are. But I can keep it as the artifact scope unchanged.

Reply:
ObjectMeta is where the object itself is saved. Scope is where the deployed object is saved. For example, your Intance object could be in the default namespace, but the pods of the instance will be deployed to the Instance.Spec.Scope namespace. With the same token, the staged artifacts should be put into the Spec.Scope namespace (just like pods), right?